cmd/poller/main.go

```diff
@@ -94,6 +95,19 @@ func gatherOptions(fs *flag.FlagSet, args ...string) options {
 	return o
 }
 
+func parseEnvPollPeriod(env string, defaultPollPeriod time.Duration) time.Duration {
+	text := os.Getenv(env)
+	if text != "" {
+		d, err := time.ParseDuration(text)
+		if err != nil {
+			logrus.WithError(err).Warn(fmt.Sprintf("invalid %s value", env))
```

Comment:
Can we add a more detailed message ?
Actual value and expected format would be great.

Reply:
Maybe using something like WithField ?

Reply:
Added some improvements, hopefully that covers it

Reply:
Looks good, thanks !